include/crm/cluster.h

```diff
@@ -33,6 +33,12 @@ extern unsigned long long crm_peer_seq;
 #define CRM_NODE_MEMBER    "member"
 
 enum crm_join_phase {
+    /* @COMPAT: crm_join_nack_quiet can be replaced by a new
+     *          crm_node_t:feature_set at a compatibility break.
+     */
```

Comment:
FYI, crm_node_t is used by multiple daemons, but the join and expected members are only used by the controller. So at a compatibility break, I plan to replace those with a void *user_data that the controller can set to a struct with those (and obviously this too). See the comment in the crm_node_t definition